src/rma/js/forms/inputs.js

```diff
@@ -5,16 +5,15 @@ import { Label } from './label';
 import { ErrorList, Wrapper } from './wrapper';
 
 
-const Input = ({ type='text', id='', name='', initial, classes=null, checked=false, onBlur, onChange, required=false }) => {
-    const classNames = classes ??`input__control input__control--${type}`;
+const Input = ({ type='text', id='', name='', initial, classes=null, checked=false, onBlur, onChange, required=false, disabled=false }) => {
     return (
         <input
             name={name}
             type={type}
             id={id}
             defaultValue={initial || ''}
             checked={checked}
-            className={classNames}
+            className={classes}
```

Comment:
why was this change reverted?

Reply:
there is no `classNames` variable in this component. I assumed that `classes` props var should be used here

Reply:
line 9:

```js
const classNames = classes ??`input__control input__control--${type}`;
```

Reply:
Oh, sorry, I accidentally removed this line than I did rebase. I'll fix it

Reply:
fixed